packages/expo-splash-screen/android/src/main/java/expo/modules/splashscreen/SplashScreenModule.kt

```diff
@@ -15,10 +15,20 @@ class SplashScreenOptions : Record {
 }
 
 class SplashScreenModule : Module() {
+  private var _userControlledAutoHideEnabled: Boolean = false
```

Comment:
```suggestion
  private var userControlledAutoHideEnabled: Boolean = false
```
Because we have proper access control in native, I'd prefer we drop the js convention of using an underscore.